src/makeRobot.js

```diff
@@ -39,6 +39,76 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    ch: version,
+    wh: wheels,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.ch}, wheels: ${this.wh}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(step = 1) {
+      if (step > 0) {
+        this.coords.y += step;
+      } 
+
+      return this;
+    },
+
+    goBack(step = 1) {
+      if (step < 0) {
+        return this;
+      } else {
+        this.coords.y = this.coords.y - step;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
    goBack(step = 1) {
      if (step > 0) {
        this.coords.y = this.coords.y - step;
      }
      
      return this;
    },
```
зміни всі методи нижче за таким принципом